backend/src/services/integration-auth/integration-sync-secret.ts

```diff
@@ -3578,7 +3578,7 @@ const syncSecretsTeamCity = async ({
       .filter((parameter) => !parameter.inherited)
       .reduce(
         (obj, secret) => {
-          const secretName = secret.name.replace(/^env\./, "");
+          const secretName = secret.name.replace("env.", "");
```

Comment:
hmmm first one only matches occurrence at the beginning. we should probably do that?